### PR TITLE
Force lowest symfony version to be 3.4

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Force lowest symfony version to be 3.4.

#### Why?

No incompatible symfony components should be installed.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -95,13 +94,10 @@
         "pulse00/ffmpeg-bundle": "^0.6",
         "symfony/browser-kit": "^3.4",
         "symfony/css-selector": "^3.4",
-        "symfony/debug": "^3.4",
         "symfony/debug-bundle": "^3.4",
         "symfony/dom-crawler": "^3.4",
         "symfony/monolog-bundle": "^3.1",
-        "symfony/monolog-bridge": "^3.4",
         "symfony/phpunit-bridge": "^3.4.8",
-        "symfony/process": "^3.4",
```
Comment (danrot) [inline composer.json]: This seems to be used in the [`bin/runtests` script](https://github.com/sulu/sulu/blob/develop/bin/runtests#L21), so it should probably stay.

File: composer.json [hunk 2]
```diff
@@ -95,13 +94,10 @@
         "pulse00/ffmpeg-bundle": "^0.6",
         "symfony/browser-kit": "^3.4",
         "symfony/css-selector": "^3.4",
-        "symfony/debug": "^3.4",
```
Comment (danrot) [inline composer.json]: This is used in the [front controllers](https://github.com/sulu/sulu/blob/develop/web/app_dev.php#L17).

Comment (alexander-schranz) [reply composer.json]: fixed

File: composer.json (hunk 1 above)
Comment (alexander-schranz) [reply composer.json]: fixed

**Pull request merged**